CONTRIBUTING.rst

```diff
@@ -515,14 +647,13 @@ these guidelines:
     useful features from newer versions.
 
 -   Adhere to guidelines for commit messages described in this `article <http://chris.beams.io/posts/git-commit/>`__.
-    This makes the lives of those who come after you a lot easier.
+    This makes the lives of those who come after you (and your future self) a lot easier.
```

Comment:
Good addition! 🤣

Reply:
Oh absolutely. People might think I write such long commit messages and docs for others, but in fact, I do it for my future solf mostly. And i saved me multiple hours, and likely days of head scratchin in the past.

Reply:
Agreed. No harm in more details!